layers/core/layer.py

```diff
@@ -5,46 +5,55 @@
 
 
 class Layer:
-    def __init__(self, init_dict={}, strict=True):
+    def __init__(self, init_data={}, strict=True):
         """
              Initialization - create a new Layer object
              :param init_dict: Optionally provide base Layer json or string
                 data on initialization
```

Comment:
This param in the pydoc comment needs to be updated to match the new param name `init_data`.

Reply:
My bad.